### ISSUE TITLE
Cannot build monky versions 2.0.0.0 and 2.0.1.0

### ISSUE THREAD

**Issue by theUncanny** [monky-hs/monky#29]
Cannot build monky 2.0.1.0 and 2.0.0.0.

Linux distribution:

```
Distributor ID: Ubuntu
Description:    Ubuntu 16.04.1 LTS
Release:    16.04
Codename:   xenial
Kernel: 4.4.0-36-generic #55-Ubuntu SMP Thu Aug 11 18:01:55 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux
```

Cabal and GHC versions:

```
# Cabal
cabal-install version 1.22.9.0
# GHC
The Glorious Glasgow Haskell Compilation System, version 7.10.3
```

> Note: Checked and failed with Cabal v1.24 and GHC v8.0.1 too.

Error message:

```
cabal: Error: some packages failed to install:
monky-2.0.1.1 failed during the building phase. The exception was:
ExitFailure 1
```

Log:

```
Configuring monky-2.0.1.0...
Building monky-2.0.1.0...
Preprocessing library monky-2.0.1.0...
[ 1 of 50] Compiling Monky.MPD        ( Monky/MPD.hs, dist/build/Monky/MPD.o )
[ 2 of 50] Compiling Monky.Outputs.Unicode ( Monky/Outputs/Unicode.hs, dist/build/Monky/Outputs/Unicode.o )
[ 3 of 50] Compiling Monky.Wifi       ( Monky/Wifi.hs, dist/build/Monky/Wifi.o )
[ 4 of 50] Compiling Monky.IP.Raw     ( dist/build/Monky/IP/Raw.hs, dist/build/Monky/IP/Raw.o )
[ 5 of 50] Compiling Monky.IP         ( Monky/IP.hs, dist/build/Monky/IP.o )
[ 6 of 50] Compiling Monky.Connectivity ( dist/build/Monky/Connectivity.hs, dist/build/Monky/Connectivity.o )
[ 7 of 50] Compiling Monky.Time       ( Monky/Time.hs, dist/build/Monky/Time.o )
[ 8 of 50] Compiling Monky.Modules    ( Monky/Modules.hs, dist/build/Monky/Modules.o )
[ 9 of 50] Compiling Monky.Examples.Utility ( Monky/Examples/Utility.hs, dist/build/Monky/Examples/Utility.o )
[10 of 50] Compiling Monky.Examples.Images ( Monky/Examples/Images.hs, dist/build/Monky/Examples/Images.o )
[11 of 50] Compiling Monky.Examples.Time ( Monky/Examples/Time.hs, dist/build/Monky/Examples/Time.o )
[12 of 50] Compiling Monky.Examples.Prepend ( Monky/Examples/Prepend.hs, dist/build/Monky/Examples/Prepend.o )
[13 of 50] Compiling Monky.Examples.Wifi ( Monky/Examples/Wifi.hs, dist/build/Monky/Examples/Wifi.o )
[14 of 50] Compiling Monky.Examples.Connectivity ( Monky/Examples/Connectivity.hs, dist/build/Monky/Examples/Connectivity.o )
[15 of 50] Compiling Monky.Examples.Combine ( Monky/Examples/Combine.hs, dist/build/Monky/Examples/Combine.o )
[16 of 50] Compiling Monky.Examples.Plain ( Monky/Examples/Plain.hs, dist/build/Monky/Examples/Plain.o )
[17 of 50] Compiling Monky.Outputs.Dzen2 ( Monky/Outputs/Dzen2.hs, dist/build/Monky/Outputs/Dzen2.o )
[18 of 50] Compiling Monky.Outputs.Ascii ( Monky/Outputs/Ascii.hs, dist/build/Monky/Outputs/Ascii.o )
[19 of 50] Compiling Monky.Outputs.Utf8 ( Monky/Outputs/Utf8.hs, dist/build/Monky/Outputs/Utf8.o )
[20 of 50] Compiling Monky.Outputs.Fallback ( Monky/Outputs/Fallback.hs, dist/build/Monky/Outputs/Fallback.o )
[21 of 50] Compiling Monky.Outputs.Show ( Monky/Outputs/Show.hs, dist/build/Monky/Outputs/Show.o )
[22 of 50] Compiling Monky.Outputs.I3 ( Monky/Outputs/I3.hs, dist/build/Monky/Outputs/I3.o )
[23 of 50] Compiling Monky.Outputs.Serialize ( Monky/Outputs/Serialize.hs, dist/build/Monky/Outputs/Serialize.o )
[24 of 50] Compiling Monky.Outputs.Guess ( Monky/Outputs/Guess.hs, dist/build/Monky/Outputs/Guess.o )

Monky/Outputs/Guess.hs:18:1: Warning:
    The import of ‘System.IO.Error’ is redundant
      except perhaps to import instances from ‘System.IO.Error’
    To import instances alone, use: import System.IO.Error()
[25 of 50] Compiling Monky.Examples.Modify ( Monky/Examples/Modify.hs, dist/build/Monky/Examples/Modify.o )
[26 of 50] Compiling Monky.Examples.MPD ( Monky/Examples/MPD.hs, dist/build/Monky/Examples/MPD.o )
[27 of 50] Compiling Monky.Utility    ( Monky/Utility.hs, dist/build/Monky/Utility.o )
[28 of 50] Compiling Monky.Disk.Common ( Monky/Disk/Common.hs, dist/build/Monky/Disk/Common.o )
[29 of 50] Compiling Monky.Disk.Btrfs ( Monky/Disk/Btrfs.hs, dist/build/Monky/Disk/Btrfs.o )
[30 of 50] Compiling Monky.CPU        ( Monky/CPU.hs, dist/build/Monky/CPU.o )
[31 of 50] Compiling Monky.Examples.CPU ( Monky/Examples/CPU.hs, dist/build/Monky/Examples/CPU.o )
[32 of 50] Compiling Monky.Memory     ( Monky/Memory.hs, dist/build/Monky/Memory.o )
[33 of 50] Compiling Monky.Examples.Memory ( Monky/Examples/Memory.hs, dist/build/Monky/Examples/Memory.o )
[34 of 50] Compiling Monky.Network.Static ( Monky/Network/Static.hs, dist/build/Monky/Network/Static.o )
[35 of 50] Compiling Monky.Network    ( Monky/Network.hs, dist/build/Monky/Network.o )
[36 of 50] Compiling Monky.Network.Dynamic ( Monky/Network/Dynamic.hs, dist/build/Monky/Network/Dynamic.o )
[37 of 50] Compiling Monky.Examples.Network ( Monky/Examples/Network.hs, dist/build/Monky/Examples/Network.o )
[38 of 50] Compiling Monky.Template   ( Monky/Template.hs, dist/build/Monky/Template.o )
[39 of 50] Compiling Monky.Alsa       ( dist/build/Monky/Alsa.hs, dist/build/Monky/Alsa.o )
[40 of 50] Compiling Monky.Examples.Alsa ( Monky/Examples/Alsa.hs, dist/build/Monky/Examples/Alsa.o )
[41 of 50] Compiling Monky.Blkid      ( Monky/Blkid.hs, dist/build/Monky/Blkid.o )
[42 of 50] Compiling Monky.Disk.Device ( Monky/Disk/Device.hs, dist/build/Monky/Disk/Device.o )
[43 of 50] Compiling Monky.Battery    ( Monky/Battery.hs, dist/build/Monky/Battery.o )
[44 of 50] Compiling Monky.Examples.Battery ( Monky/Examples/Battery.hs, dist/build/Monky/Examples/Battery.o )
[45 of 50] Compiling Monky.Examples.File ( Monky/Examples/File.hs, dist/build/Monky/Examples/File.o )
[46 of 50] Compiling Monky.VersionTH  ( Monky/VersionTH.hs, dist/build/Monky/VersionTH.o )
[47 of 50] Compiling Monky.Version    ( Monky/Version.hs, dist/build/Monky/Version.o )
[48 of 50] Compiling Monky            ( Monky.hs, dist/build/Monky.o )
[49 of 50] Compiling Monky.Disk       ( Monky/Disk.hs, dist/build/Monky/Disk.o )
[50 of 50] Compiling Monky.Examples.Disk ( Monky/Examples/Disk.hs, dist/build/Monky/Examples/Disk.o )
In-place registering monky-2.0.1.0...
Preprocessing executable 'monky' for monky-2.0.1.0...
[1 of 1] Compiling Main             ( bin/Main.hs, dist/build/monky/monky-tmp/Main.o )

bin/Main.hs:92:32:
    Not in scope: ‘<>’
    Perhaps you meant one of these:
      ‘<$>’ (imported from Options.Applicative),
      ‘<*>’ (imported from Options.Applicative),
      ‘*>’ (imported from Options.Applicative)

bin/Main.hs:92:95:
    Not in scope: ‘<>’
    Perhaps you meant one of these:
      ‘<$>’ (imported from Options.Applicative),
      ‘<*>’ (imported from Options.Applicative),
      ‘*>’ (imported from Options.Applicative)

bin/Main.hs:92:108:
    Not in scope: ‘<>’
    Perhaps you meant one of these:
      ‘<$>’ (imported from Options.Applicative),
      ‘<*>’ (imported from Options.Applicative),
      ‘*>’ (imported from Options.Applicative)

bin/Main.hs:93:32:
    Not in scope: ‘<>’
    Perhaps you meant one of these:
      ‘<$>’ (imported from Options.Applicative),
      ‘<*>’ (imported from Options.Applicative),
      ‘*>’ (imported from Options.Applicative)

bin/Main.hs:93:89:
    Not in scope: ‘<>’
    Perhaps you meant one of these:
      ‘<$>’ (imported from Options.Applicative),
      ‘<*>’ (imported from Options.Applicative),
      ‘*>’ (imported from Options.Applicative)

bin/Main.hs:93:102:
    Not in scope: ‘<>’
    Perhaps you meant one of these:
      ‘<$>’ (imported from Options.Applicative),
      ‘<*>’ (imported from Options.Applicative),
      ‘*>’ (imported from Options.Applicative)

bin/Main.hs:94:27:
    Not in scope: ‘<>’
    Perhaps you meant one of these:
      ‘<$>’ (imported from Options.Applicative),
      ‘<*>’ (imported from Options.Applicative),
      ‘*>’ (imported from Options.Applicative)

bin/Main.hs:94:94:
    Not in scope: ‘<>’
    Perhaps you meant one of these:
      ‘<$>’ (imported from Options.Applicative),
      ‘<*>’ (imported from Options.Applicative),
      ‘*>’ (imported from Options.Applicative)

bin/Main.hs:95:30:
    Not in scope: ‘<>’
    Perhaps you meant one of these:
      ‘<$>’ (imported from Options.Applicative),
      ‘<*>’ (imported from Options.Applicative),
      ‘*>’ (imported from Options.Applicative)

bin/Main.hs:95:77:
    Not in scope: ‘<>’
    Perhaps you meant one of these:
      ‘<$>’ (imported from Options.Applicative),
      ‘<*>’ (imported from Options.Applicative),
      ‘*>’ (imported from Options.Applicative)

bin/Main.hs:96:27:
    Not in scope: ‘<>’
    Perhaps you meant one of these:
      ‘<$>’ (imported from Options.Applicative),
      ‘<*>’ (imported from Options.Applicative),
      ‘*>’ (imported from Options.Applicative)

bin/Main.hs:97:36:
    Not in scope: ‘<>’
    Perhaps you meant one of these:
      ‘<$>’ (imported from Options.Applicative),
      ‘<*>’ (imported from Options.Applicative),
      ‘*>’ (imported from Options.Applicative)

bin/Main.hs:98:39:
    Not in scope: ‘<>’
    Perhaps you meant one of these:
      ‘<$>’ (imported from Options.Applicative),
      ‘<*>’ (imported from Options.Applicative),
      ‘*>’ (imported from Options.Applicative)

bin/Main.hs:114:31:
    Not in scope: ‘<>’
    Perhaps you meant one of these:
      ‘<$>’ (imported from Options.Applicative),
      ‘<*>’ (imported from Options.Applicative),
      ‘*>’ (imported from Options.Applicative)
```

**Comment by Ongy** [comment]
Should be fixed on master (optparse-applicative-0.13 removed a reexport).
Can you please test and report back.

**Comment by Ongy** [comment]
2.1.0 builds and is uploaded to hackage.
